docs/en/latest/plugins/opentelemetry.md

```diff
@@ -23,9 +23,9 @@ title: opentelemetry
 
 ## Description
 
-[OpenTelemetry](https://opentelemetry.io/) report Tracing data according to [opentelemetry specification](https://github.com/open-telemetry/opentelemetry-specification).
+[OpenTelemetry](https://opentelemetry.io/) report Tracing data according to [OpenTelemetry specification](https://opentelemetry.io/docs/reference/specification/).
 
-Just support reporting in `HTTP` with `Content-Type=application/x-protobuf`, the specification: [OTLP/HTTP Request](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp-request)。
+Just support reporting in `HTTP` with `Content-Type=application/x-protobuf`, the specification: [OTLP/HTTP Request](https://opentelemetry.io/docs/reference/specification/protocol/otlp/#otlphttp-request).
```

Comment:
Note that this sentence doesn't really make sense grammatically. (I chose to only update the link. I'll let someone else rework the sentence if you feel that is necessary.

Reply:
Thanks for your suggestion, would you like to have it a try ?

Reply:
Sure I can give it a try tomorrow.